### ISSUE TITLE
UX Issues: New App from Fresh Install

### ISSUE THREAD

**Issue by ricklove** [rvion/CushyStudio#85]
This is an amazing project! I'll try to document some issues to help it be successful:

### The examples are broken:

![image](https://github.com/rvion/CushyStudio/assets/703880/5295e22f-ce56-408c-92ff-ff4f84223cf5)

### New App: Does not seem to setup correctly or seems broken

![image](https://github.com/rvion/CushyStudio/assets/703880/89ead1c4-40f9-4c6a-b5f7-653e510c8d06)

### New App: Git error before publishing git repo to origin

![image](https://github.com/rvion/CushyStudio/assets/703880/a3db004b-2ca4-45ec-814f-f2b907d00bdd)

**Comment by rvion** [comment]
Yes, I'm sorry cushy is in this state, I broke master a bit, and due to time contraints, haven't fixed it yet. It will probably remain like that for a few days, I can only hope you'll give it an other go in a few days.

Boring stuff are happening right now:  

- the whole storage engine is being rewritten using a production ready database; no more Json everywhere. No more need to reset your database every once in a while.

- the new app "outputs" concept is being made a foundational piece of cushy, upgrading the core abstraction a bit. Apps can output images, comfy workflows, comfy prompts, text, 3d scene, gaussian splats, vidéo, 2d collage with images overlayed, etc. All those outputs are getting a dedicated table in the db, and will all be displayed in the new...

- unified gallery, that does not only show images or videos, but anything an app produces

Due to only being able to work on evenings and weekend, and since I had few users, I took the liberty to break things. Stability will come very soon , 

thanks again for your bug report and your patience
